src/shared/components/Orchestrator/Orchestrator.tsx

```diff
@@ -82,6 +83,7 @@ export function Orchestrator(props: OrchestratorProps) {
   const { source, surveyUnitData, getReferentiel, mode, metadata } = props
 
   const initialCurrentPage = surveyUnitData?.stateData?.currentPage
+  const currentState = surveyUnitData?.stateData?.state
```

Comment:
you could rename 'currentState' to 'initialState' for coherence with 'initialCurrentPage'